### DISCUSSION TITLE
Cannot pass SSL context to asyncpg from `create_async_engine` in SQLAlchemy 1.4

### DISCUSSION THREAD

**Discussion by connebs** [sqlalchemy/sqlalchemy#5975] [Usage Questions] [answered]
**Describe the bug**
Some PostgreSQL servers are configured with custom root certs, etc. so sometimes you need to do a little bit of extra legwork to configure SSL.

`asyncpg` allows you to do this by passing an `ssl.SSLContext` to the `ssl` kwarg on `asyncpg.create_pool`, as described in this issue: https://github.com/MagicStack/asyncpg/issues/238.

With SQLAlchemy 1.4's async support, obviously it would be helpful if this context could be passed through to the `asyncpg` driver, probably when creating an async engine with `create_async_engine`.

I had a dig through the SQLA14 source and couldn't find any kwarg-passing etc. that allows this behavior.

**Expected behavior**
```py
my_ssl_ctx = ssl.create_default_context(cafile=PATH_TO_CA_FILE)
my_ssl_ctx.verify_mode = ssl.CERT_REQUIRED

async_engine = create_async_engine(POSTGRES_URL, ssl=my_ssl_context)
```

**To Reproduce**
Try to use SQLAlchemy 14 async engine to connect to a PostgreSQL server with custom root/intermediate certs. You should get an error. For example Amazon's RDS.

**Versions.**
 - Python: 3.8.2
 - SQLAlchemy: 1.4.0-beta.3
 - Database: PostgreSQL
 - DBAPI: asyncpg

**Have a nice day!**

**Comment by zzzeek** [comment, marked as answer]
hi - 

have you looked at [connect_args](https://docs.sqlalchemy.org/en/14/core/engines.html#use-the-connect-args-dictionary-parameter)  ?   this works the same way with the async engine:


    my_ssl_ctx = ssl.create_default_context(cafile=PATH_TO_CA_FILE)
    my_ssl_ctx.verify_mode = ssl.CERT_REQUIRED

    async_engine = create_async_engine(POSTGRES_URL, connect_args={"ssl": my_ssl_context})

**Reply by connebs** [reply]
That was exactly what I was looking for, don't know how I missed it.

Thanks!
